extensions/KHR/EGL_KHR_platform_wayland.txt

```diff
@@ -93,6 +94,21 @@ New Behavior
     until the next buffer swap. The rationale behind this behavior is to keep
     operations result accurate until the next swap.
 
+    Synchronously within the call to eglSwapBuffers the EGL implementation will
+    invoke a series of requests on the associated wl_surface, including
+    attaching the new front buffer, a full surface damage via wl_surface.damage
+    or wl_surface.damage_buffer, and finally a wl_surface.commit.
+
+    If the EGL implementation associated a wp_linux_drm_syncobj_surface_v1
+    object with the surface, an acquire and a release point will be associated
+    with the surface prior to the commit as well.
+
+    If KHR_swap_buffers_with_damage is available, the equivalent to calling
```

Comment:
This section shouldn't really be necessary. eglSwapBuffersWithDamageKHR() already states that it inherits all the behavior of eglSwapBuffers():

    As an alternative to eglSwapBuffers use:

        EGLBoolean eglSwapBuffersWithDamageKHR (
            EGLDisplay dpy,
            EGLSurface surface,
            const EGLint *rects,
            EGLint n_rects);

        to do the same thing as eglSwapBuffers but additionally report
        a list of rectangles that define the region that has truly
        changed since the last frame.

I think after applying the above suggestion about properly inserting this language into the eglSwapBuffers() portion of the spec, this inheritance would be more naturally implied in readers' minds.